crates/ruff/resources/test/fixtures/pycodestyle/W19.py

```diff
@@ -0,0 +1,145 @@
+#: W191
+if False:
+	print  # indented with 1 tab
+#:
+
+
+#: W191
+y = x == 2 \
+	or x == 3
+#: E101 W191 W504
+if (
+        x == (
+            3
+        ) or
+        y == 4):
+	pass
+#: E101 W191
+if x == 2 \
+    or y > 1 \
+        or x == 3:
+	pass
+#: E101 W191
+if x == 2 \
+        or y > 1 \
+        or x == 3:
+	pass
+#:
+
+#: E101 W191 W504
+if (foo == bar and
```

Comment:
I'm using the test as is from pycodestyle (https://github.com/PyCQA/pycodestyle/blob/main/testsuite/W19.py#L31) should I fix this typo to silence the spell checker?

Reply:
Yeah maybe just change `frop` to `bop`?